pennylane_lightning/src/util/Kokkos_Sparse.hpp

```diff
@@ -155,12 +156,25 @@ void apply_Sparse_Matrix_Kokkos(
 #else
 constexpr bool USE_KOKKOS = false;
 namespace Pennylane::Util {
+
 /**
  * @brief Apply a sparse matrix to a vector with Kokkos.
- *  Abort, because there is no Kokkos and Kokkos Kernels installation.
+ *  The only purpose of this function is to throw an exception if there is no
+ *  Kokkos and Kokkos Kernels installation, except if throw_exception is false.
  *
- * @tparam fp_precision
- * @return std::vector<data_type<fp_precision>>
+ * @tparam fp_precision data float point precision.
+ * @tparam index_type integer type used as indices of the sparse matrix.
+ * @param vector_ptr    pointer to the vector.
+ * @param vector_size   size of the vector.
+ * @param row_map_ptr   pointer to the row_map.
+ *                      The j element encodes the number of non-zeros above row
+ * j.
+ * @param row_map_size  number of elements in the row_map.
+ * @param entries_ptr   pointer to the column indices of the non-zero elements.
+ * @param values_ptr    non-zero elements.
+ * @param numNNZ        number of non-zero elements.
+ * @param result        result of the matrix vector multiplication.
+ * @param throw_exception if will true an exception.
```

Comment:
```suggestion
 * @param throw_exception throws exception if true.
```

Though, I still think this as an argument does not make much sense.